internal/generator/vector/output/cloudwatch/cloudwatch.go

```diff
@@ -158,6 +176,64 @@ del(.source_type)
 	}
 }
 
+func GroupNameTemplateRemap(componentID string, inputs []string, groupName string) Element {
```

Comment:
It should be possible to make this "generic" for any place we wish to template "internal/generators/vector/outputs/common" ... maybe common/template ?  groupName here is the templated field